packages/babel-types/README.md

```diff
@@ -1432,6 +1432,19 @@ Aliases: `Flow`, `UserWhitespacable`
 
 ---
 
+### objectTypeSpreadProperty
```

Comment:
I took a diff:
```
git diff "526a7d20ef1f6b12bccac0f500bcb8522c90ac00^" 526a7d20ef1f6b12bccac0f500bcb8522c90ac00 > create-class.patch
```
and applied the resulting patch to 6.x branch, where 526a7d20ef1f6b12bccac0f500bcb8522c90ac00 is commit that merged #5554, that's all